Minor Fixes from Dry Run (#336)
AY2021S1-CS2103T-F11-4/tp: docs/UserGuide.md

 
     ![Result Display slowing error message](images/command-screenshots/result_display_error_syntax.png)
     

> * **Display Panel** shall render *Client Cards* or *Country Cards* when relevant commands are sent. Initially it will show you your local time to the precise second, as well as your current location.

Reviewer: ```suggestion
* **Display Panel** shall render *Client Cards* or *Country Cards* when relevant commands are sent. The default view shows you your local time to the precise second, as well as your current location.
```
